SwiftGRPC-Carthage.xcodeproj/GeneratedModuleMap/CNIONghttp2/module.modulemap

```diff
@@ -1,4 +1,4 @@
 module CNIONghttp2 {
-    umbrella "/Users/mrebello/Development/grpc-swift/.build/checkouts/swift-nio-http2.git-1684232237084789971/Sources/CNIONghttp2/include"
```

Comment:
I'm a bit surprised by all the `.git-1684232237084789971` suffixes disappearing. I'd expect the values to have changed, but that there would still be a suffix present. Which Swift version did you use to generate the new project? (I think we should stick with Swift 4.2 for now, to ensure that we don't break the build for Swift 4.2 users by generating the project with Swift 5.) See https://medium.com/xcblog/switching-swift-versions-inside-xcode-using-toolchains-755b28831c43 on selecting the Swift 4.2 toolchain.

Reply:
Of course: I'm using swift 5. I'll fix that

Reply:
Done